karma.conf.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path')
+const webpackConfig = require('./webpack.config.js')
+
+const IS_TRAVIS = process.env.TRAVIS
```

Comment:
We need to vary behavior slightly for the travis/local test runs, se we check for the presence of the `TRAVIS` environment variable. Travis sets this to the string `true`.